public/js/controllers/index.js

```diff
@@ -15,6 +15,7 @@ angular.module('mean.system').controller('IndexController', [
       game.joinGame();
       $location.path('/app');
     };
+    $scope.name = "ello";
```

Comment:
This line doesn't seem to be useful to be used anywhere.
Can it be possibly removed?